File: 0118-pascals-triangle/0118-pascals-triangle.cpp
class Solution {
public:
    vector<int> generaterow(int row){
        long long ans = 1;
        vector<int> ansroww;
        ansroww.push_back(ans);
        for(int col=1;col<row;col++){
            ans = ans*(row-col);
            ans = ans/col;
            ansroww.push_back(ans);
        }
        return ansroww;
    }
    vector<vector<int>> generate(int numRows) {
        vector<vector<int>> ans;
        for(int i=1; i<=numRows;i++){
            ans.push_back(generaterow(i));
        }
        return ans;
    }
};

File: 0118-pascals-triangle/0118-pascals-triangle.py

class Solution:
    def generaterow(self, row):
        ans = 1
        ansroww = []
        ansroww.append(ans)

        for col in range(1, row):
            ans = ans * (row - col)
            ans = ans // col
            ansroww.append(ans)

        return ansroww

    def generate(self, numRows):
        ans = []

        for i in range(1, numRows + 1):
            ans.append(self.generaterow(i))

        return ans

